clustered/client/src/test/java/org/ehcache/clustered/client/BasicClusteredCacheTest.java

```diff
@@ -233,6 +233,25 @@ public void testLargeValues() throws Exception {
     }
   }
 
+  @Test
+  public void testClusteredCacheForcingChainResolution() throws Exception {
+    final CacheManagerBuilder<PersistentCacheManager> clusteredCacheManagerBuilder =
+      newCacheManagerBuilder()
+        .with(cluster(CLUSTER_URI).autoCreate())
+        .withCache("clustered-cache", newCacheConfigurationBuilder(Long.class, String.class,
+          ResourcePoolsBuilder.newResourcePoolsBuilder()
+            .with(clusteredDedicated("primary-server-resource", 2, MemoryUnit.MB)))
+          .add(new ClusteredStoreConfiguration(Consistency.STRONG)));
+
+    try (PersistentCacheManager cacheManager = clusteredCacheManagerBuilder.build(true)) {
```

Comment:
I don't get it. How do you test it is forcing chain resolution?

Reply:
This shouldn't still be here.  The test in CommonServerStoreProxyTest covers this.